js/models/profile/Profile.js

```diff
@@ -13,7 +13,7 @@ export default class extends BaseModel {
       handle: '',
       location: '',
       moderator: false,
-      name: `ob ${Math.random().toString(36).slice(2)}`,
+      name: `OB ${Math.random().toString(36).slice(1).substring(1,7)}`,
```

Comment:
A more robust way to do it would be:
```
`OB ${(Math.random() * 1000000000).toString(36).split('.')[0]}`
```
That should provide a 6 character random string, with no period.